exporter/awsemfexporter/datapoint.go

```diff
@@ -156,6 +164,92 @@ func (dps histogramDataPointSlice) CalculateDeltaDatapoints(i int, instrumentati
 	}}, true
 }
 
+// CalculateDeltaDatapoints retrieves the ExponentialHistogramDataPoint at the given index.
+func (dps exponentialHistogramDataPointSlice) CalculateDeltaDatapoints(i int, instrumentationScopeName string, detailedMetrics bool) ([]dataPoint, bool) {
+	metric := dps.ExponentialHistogramDataPointSlice.At(i)
```

Comment:
```suggestion
func (dps exponentialHistogramDataPointSlice) CalculateDeltaDatapoints(idx int, instrumentationScopeName string, detailedMetrics bool) ([]dataPoint, bool) {
	metric := dps.ExponentialHistogramDataPointSlice.At(idx)
```

`i` is shadowed later in for loops.  I believe this is the only place the method argument is expected to be used, so it's probably easiest to just change its name here.

Reply:
Fixed.